hg-start.cmd

```diff
@@ -0,0 +1,9 @@
+@echo off
```

Comment:
These scripts could be moved into the Apps folder to keep the folder structure cleaner.

Reply:
Moved